operator/v1/types_network.go

```diff
@@ -100,12 +100,79 @@ type DefaultNetworkDefinition struct {
 	KuryrConfig *KuryrConfig `json:"kuryrConfig,omitempty"`
 }
 
+// SimpleMacvlanConfig contains configurations for macvlan interface.
+type SimpleMacvlanConfig struct {
+	// master is the host interface to create the macvlan interface from.
+	// If not specified, it will be default route interface
+	// +optional
+	Master string `json:"master,omitempty"`
+
+	// IpamConfig configures IPAM module will be used for IP Address Management(IPAM).
+	IpamConfig IpamConfig `json:"ipamConfig"`
```

Comment:
golang style is that acronyms should always be either all-uppercase or all-lowercase. So `ipamConfig` is correct for the JSON tag, but the variable and the type name should be `IPAMConfig`. Likewise `StaticIPAMAddresses`, `DNS StaticIPAMDNS`, `IPAMTypeDHCP`, etc.

Reply:
fixed. PTAL?